### PR TITLE
added work and break minute displays and settings; added functionalit…

### PR DESCRIPTION
**Pull request by janessatran**
- added work/break minute settings display and buttons 
- added functionality to play/pause buttons

I need help with making the code more modular. There is a lot of repetition currently 😳.

### CODE REVIEW

File: timer.js [hunk 1]
```diff
@@ -42,5 +49,124 @@ function stop() {
...
+var secondsDisplay = document.getElementById('seconds');
+var timer; 
+var minutes; 
+var seconds = secondsDisplay.textContent;;
+var intervalId;
+var intervalCount = -1;
+
+// add event listeners for play button
+var playButton = document.getElementById('play');
+if (playButton.addEventListener) {
```
Comment (botanical) [inline timer.js:79]: what does the conditional check? That the button wants a listener? 😮 🤔

Comment (janessatran) [reply timer.js:79]: A conditional check that you can use that method for the browser:

"You can also add event listeners using a method called addEventListener. However, this method isn't supported in Internet Explorer 8, so if you use this method, you need to add some conditional functions to check for browser functionality before running the function."

https://idratherbewriting.com/events-and-listeners-javascript/

:)

Comment (botanical) [reply timer.js:79]: Ooh thanks, B!

**Review by botanical — APPROVED** [review]
👍  looks good to me! Have you tested it locally?

**Pull request merged**